BlueShift-iOS-SDK/InApps/BlueShiftNotificationWebViewController.m

```diff
@@ -219,7 +219,13 @@ - (CGRect)positionWebView{
     frame.size = size;
     webView.autoresizingMask = UIViewAutoresizingNone;
     
-    CGSize screenSize = self.window.bounds.size;
+    CGSize screenSize = CGSizeZero;
+    if (self.window) {
+        screenSize = self.window.bounds.size;
+    } else {
+        screenSize = [[UIScreen mainScreen] bounds].size;
+    }
```

Comment:
code can be optimized if we move this code to a utility/helper method as mentioned above